includes/class-wp-notify-demo.php

```diff
@@ -0,0 +1,249 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * @package wp-feature-notifications
+ *
+ * Development demo files
+ **/
+
```

Comment:
It's something I'm planning on taking on as a larger project, but what do you think about using a namespace for this instead of prefixes? 
Probably

```php
namespace WP\Notifications\Demo;
use WP_Admin_Bar;
```
etc